keyserver/handler.go

```diff
@@ -56,3 +65,12 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 		TB:   tb,
 	}, nil
 }
+
+func (server *ConiksServer) StoreUserPoliciesToKV(up *p.ConiksUserPolicies) error {
```

Comment:
As I suggested in https://github.com/coniks-sys/coniks-go/pull/29#discussion_r72646673, if we store the user policies as part of the user leaf node's ``value()``, I don't think we need a separate ``StoreUserPoliciesToKV()``.